core/src/qos_service.rs

```diff
@@ -9,58 +9,67 @@ use {
         cost_model::{CostModel, TransactionCost},
         cost_tracker::CostTrackerError,
     },
-    solana_sdk::transaction::{self, SanitizedTransaction, TransactionError},
-    std::sync::{Arc, RwLock},
+    solana_sdk::{
+        timing::AtomicInterval,
+        transaction::{self, SanitizedTransaction, TransactionError},
+    },
+    std::{
+        sync::atomic::{AtomicBool, AtomicU64, Ordering},
+        sync::{Arc, RwLock},
```

Comment:
```suggestion
        sync::{atomic::{AtomicBool, AtomicU64, Ordering}, Arc, RwLock},
```

Reply:
Not sure if that's formatted right, though

Reply:
Cool, thanks! Will let `cargo fmt` handle it